src/nnue/nnue_feature_transformer.h

```diff
@@ -154,7 +160,18 @@ namespace Eval::NNUE {
       for (IndexType p = 0; p < 2; ++p) {
         const IndexType offset = kHalfDimensions * p;
 
-  #if defined(USE_AVX2)
+  #if defined(USE_AVX512)
+        auto out = reinterpret_cast<__m512i*>(&output[offset]);
+        for (IndexType j = 0; j < kNumChunks; ++j) {
+          __m512i sum0 = _mm512_load_si512(
+              &reinterpret_cast<const __m512i*>(accumulation[perspectives[p]][0])[j * 2 + 0]);
```

Comment:
The indentations here are inconsistent.

Reply:
I changed the indentation of like 171, don't really know whether it should be 2 or 4 spaces here because it differs from line to line in this file

Reply:
Line 167 and 169 should be the same but aren't.  I'm not sure about 2 or 4 spaces for the same reason as you.